
Showing logs for build [32mmeta-seldonio-seldon-core-pr-19-znc7r-3[0m stage [32mmeta-pipeline[0m and container [32mstep-credential-initializer-cknq7[0m
{"level":"warn","ts":1592391221.4587982,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1592391221.459396,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mmeta-seldonio-seldon-core-pr-19-znc7r-3[0m stage [32mmeta-pipeline[0m and container [32mstep-working-dir-initializer-bns9q[0m
{"level":"warn","ts":1592391222.542507,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1592391222.545381,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mmeta-seldonio-seldon-core-pr-19-znc7r-3[0m stage [32mmeta-pipeline[0m and container [32mstep-place-tools[0m

Showing logs for build [32mmeta-seldonio-seldon-core-pr-19-znc7r-3[0m stage [32mmeta-pipeline[0m and container [32mstep-git-source-meta-seldonio-seldon-core-pr-19-hd82n[0m
{"level":"warn","ts":1592391229.249498,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1592391233.4514844,"logger":"fallback-logger","caller":"git/git.go:103","msg":"Successfully cloned https://github.com/SeldonIO/seldon-core.git @ 26d9eefc119b36e3620dd379117c9088f308e5fa in path /workspace/source"}

Showing logs for build [32mmeta-seldonio-seldon-core-pr-19-znc7r-3[0m stage [32mmeta-pipeline[0m and container [32mstep-git-merge[0m
Using SHAs from PULL_REFS=master:26d9eefc119b36e3620dd379117c9088f308e5fa,1966:add5d2b626529a5f87c5a5e1212708fba560548e
DEBUG: ran git fetch --unshallow origin add5d2b626529a5f87c5a5e1212708fba560548e: 26d9eefc119b36e3620dd379117c9088f308e5fa: in 
DEBUG: ran git checkout master in 
DEBUG: ran git reset --hard 26d9eefc119b36e3620dd379117c9088f308e5fa in 
DEBUG: ran clean --force -d . in 
error: error during merge: merging add5d2b626529a5f87c5a5e1212708fba560548e into master: git output: Auto-merging .github/ISSUE_TEMPLATE/bug_report.md
CONFLICT (content): Merge conflict in .github/ISSUE_TEMPLATE/bug_report.md
Automatic merge failed; fix conflicts and then commit the result.: failed to run 'git merge add5d2b626529a5f87c5a5e1212708fba560548e' command in directory '', output: 'Auto-merging .github/ISSUE_TEMPLATE/bug_report.md
CONFLICT (content): Merge conflict in .github/ISSUE_TEMPLATE/bug_report.md
Automatic merge failed; fix conflicts and then commit the result.'
[31m
Pipeline failed on stage 'meta-pipeline' : container 'step-git-merge'. The execution of the pipeline has stopped.[0m
